load_db_from_json_file_mysql.sh

```
@@ -0,0 +1,3 @@
+python manage.py flush --no-input
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)